src/rum/core.cljs

```diff
@@ -98,7 +103,14 @@
            (fn []
              (this-as this
                (let [state @(state this)]
-                 (clj->js (transduce (map #(% state)) merge {} child-context))))))}
+                 (clj->js (transduce (map #(% state)) merge {} child-context))))))
+         :contextTypes
+         (when-not (empty? context-types)
+           (clj->js (apply merge context-types)))
+         :childContextTypes
+         (when-not (empty? child-context-types)
+           (clj->js (apply merge child-context-types)))}
+
```

Comment:
I think you are right, it should probably throw in there. If the user really wants to override  the context types put there by mixins, he can still do it with a `{:class-properties {:contextTypes ... }}` map that will override everything.

Also, thanks a lot for the derivatives library!